build_tools/rocksdb-lego-determinator

```diff
@@ -379,23 +379,6 @@ LITE_BUILD_COMMANDS="[
     }
 ]"
 
-#
-# Report RocksDB lite binary size to scuba
-REPORT_LITE_BINARY_SIZE_COMMANDS="[
-    {
-        'name':'Rocksdb Lite Binary Size',
-        'oncall':'$ONCALL',
-        'executeLocal': 'true',
-        'steps': [
-            $CLEANUP_ENV,
-            {
-                'name':'Report RocksDB Lite binary size to scuba',
-                'shell':'cd $WORKING_DIR; tools/report_lite_binary_size.sh',
-                'user':'root',
-            },
-        ],
-]"
-
```

Comment:
I'm assuming we don't have/need such a job anymore, right?

Reply:
Yeah, the test is never been used. Also our regression test is reporting binary size now, including LITE, shared, static: https://fburl.com/unidash/b9ana9qd